deploy/crds/kieapp.crd.yaml

```diff
@@ -307,6 +307,19 @@ spec:
                     description: The password to use for keystore generation.
                     format: password
                     type: string
+                  startupStrategy:
+                    description: Startup strategy for Console and Kieserver
+                    properties:
+                      controllerTemplateCacheTTL:
+                        description: Controller Template Cache TTL to use when the
+                          ControllerBasedStartupStrategy is choosed and BC is deployed,
+                          default is 5000
+                        type: integer
+                      strategyName:
+                        description: StartupStrategy to use. Default is OpenShiftStartupStrategy,
+                          option is ControllerBasedStartupStrategy.
```

Comment:
can we improve this, option is....

Reply:
ok, improved with description form image.yaml